src/open_inwoner/accounts/forms.py

```diff
@@ -56,33 +61,46 @@ def clean_email(self):
         raise ValidationError(_("This user has been deactivated"))
 
 
-class UserForm(forms.ModelForm):
+class BaseUserForm(forms.ModelForm):
     class Meta:
         model = User
         fields = (
-            "first_name",
-            "last_name",
             "display_name",
             "email",
             "phonenumber",
-            "birthday",
-            "street",
-            "housenumber",
-            "postcode",
-            "city",
+            "image",
         )
 
+    def __init__(self, user, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields["image"].help_text = None
+        if user.contact_type != ContactTypeChoices.begeleider:
+            self.fields["image"].widget = forms.HiddenInput()
```

Comment:
This feels unusual, why kill the help_text, and why the hidden field? Shouldn't we just `del self.fields["image"]`?